code/game/machinery/computer/message.dm

```diff
@@ -34,6 +34,23 @@
 
 	light_color = LIGHT_COLOR_DARKGREEN
 
+/obj/machinery/computer/message_monitor/Initialize()
+	..()
```

Comment:
```suggestion
	. = ..()
```

Reply:
No need for the `.` there I think, since it's returning `INITIALIZE_HINT_LATELOAD` instead on the next line.

Reply:
Oh yeah you're right.